### ISSUE TITLE
Potential mining fees issue. Somewhat repeatable.

### ISSUE THREAD

**Issue by ifarnung** [bisq-network/bisq#3249]
So I usually wait until the mempool is light to make orders with the goal being 4087 sats + fees in BSQ which is the lowest I've been able to get.
And I keep noticing an inconsistency that may or may not be an issue.
Example: I create a new offer to Sell Bitcoin and see that the funds needed for mining are 4087 sats. Screenshot:
![image](https://user-images.githubusercontent.com/55181240/64766040-059f4480-d54e-11e9-8318-ba3d2965a2b9.png)
Ok, I create that offer and then go to create an identical offer and now the mining fees have jumped (and I felt that it was happening everytime I created a second offer, though it could just be coincidence that fees jumped in that few seconds between order creation)
Screenshot: 
![image](https://user-images.githubusercontent.com/55181240/64766208-5151ee00-d54e-11e9-8b97-473c14bb6d1d.png)
But then to test it/doublecheck it, I go to create a Buy order and the mining fees are 4087 again. 
Screenshot:
![image](https://user-images.githubusercontent.com/55181240/64766451-ac83e080-d54e-11e9-9208-f56d47ed176e.png)

And right now, I created another Buy order and when I went to create another, I was able to see 4087 sats in mining fees again.

So it appears to me (and I might be not seeing something right) that when I go to create a second Sell order, the mining fees jump.

**Comment by ifarnung** [comment]
Just to follow up, I reproduced the exact issue right now while creating two SELL orders.  First one, 4087 sats.  Second one, a few clicks later. Here's a screenshot. and I recognize 5547 as a common step in sats.
![image](https://user-images.githubusercontent.com/55181240/64783664-0c8b7e80-d571-11e9-9be2-5369337fac5a.png)

Thoughts?

**Comment by ifarnung** [comment]
Still able to repeat the issue today.  Mining fees jumped to 5557 on the second sell order created.
Restarting didn't change that.  But I can create a buy order and it shows mining fees of 4087.

**Comment by wiz** [comment]
It sounds like the fees are jumping between 1 and 2 sats per byte.

**Comment by ifarnung** [comment]
I was going to try to better explain the issue I'm having as for me it's repeatable every time I make a second order, but @Wiz and I did some real time troubleshooting with him trying to replicate the issue on his end, and he was able to figure out why it was happening.

While an order that has just been created isn't confirmed on-chain, any subsequent orders will use a higher fee estimate than the initial order.  So this is why the fee on every second order I created would jump even if the mempool was similarly full. 

So now, I can just wait until the order has some confirmations (I think 2 or 3 until the fee goes back to the minimum estimate) and then create a new order.  At least for me, this is a better way to manage blockspace usage.

Thanks @wiz for your help and I will close this issue.
